src/styles/index.scss

```diff
@@ -1,3 +1,19 @@
+@use './mixins' as *;
+@use './variables' as *;
+@use './card' as *;
+@use './stars' as *;
+@use './price' as *;
+@use './button' as *;
+
 body {
+  @include center;
+  @include roboto400;
+
+  border: 1px;
+  border-color: #f3f3f3;
+  border-radius: 5px;
   margin: 0;
+  width: 198px;
+  height: 406px;
```

Comment:
[checklist](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules): Don't fix container size (if there is no such a requirement)
Let the content size dictate it. To avoid overflow or accidental scroll bar